uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-notification-badge.component.js

```diff
@@ -37,7 +37,7 @@ define([
     function fetchCount () {
       vm.loading.count = true;
 
-      return LeaveRequest.all(vm.filters)
+      return LeaveRequest.all(vm.filters, null, null, null, false)
```

Comment:
Is that just to prevent caching? Shall we start thinking of refactoring a model? At least this is not a Reqangular model, so no extra PR's are needed, but this looks ugly `(vm.filters, null, null, null, false)`

I would prefer:
```js
{ filters: vm.filters, cache: false }
```
instead. Please note, I am *not* asking you to refactor right now, however, I would still like to point it out. I remember we had such conversations in the past but now it really looks like a disadvantage of a fixed params listing in this case.

cc @AkA84